content/03.heterogeneity.md

```diff
@@ -1,16 +1,20 @@
+In rare diseases, the ability to get an enormous number of measurements from a vanishingly small number of samples using high-throughput methods is both the upside and the downfall of these methods. 
+These ‘omic’ methods generate highly dimensional data - that is, data with many features (e.g., all of the mRNA transcripts in a sample). 
+Perhaps counterintuitively, more feature-rich data can make a prediction problem more challenging. 
+This is because statistical interrogation of a large number of measurements requires an abundance of samples or observations, which is often not the case in rare disease. 
+This lack of samples gives rise to the “curse of dimensionality” (i.e., few samples but many features), which can be a major impediment in analyzing feature-rich data in sample-deficient contexts such as rare disease [@doi:10.1038/nrc2294] 
+In particular, increased numbers of features results in increased sparsity (missing observations), more dissimilarity between samples, and increased redundancy between individual features or combinations of features [@doi:10.1038/s41592-018-0019-x]; the consequence of this additional data is a more challenging prediction problem, rather than an easier one. 
+Furthermore, rare disease data collection and aggregation methods can add to these challenges by introducing technical variability into the data at hand.
 In this section, we will discuss strategies like simplifying data and addressing technical artifacts through dimension reduction  which can help mitigate these challenges. 
 Dimensionality reduction methods including unsupervised approaches like multidimensional scaling (MDS), principal components analysis (PCA), t-distributed stochastic neighbor embedding (t-SNE), and uniform manifold approximation and projection (UMAP) [@doi:10.1007/978-3-540-33037-0; @doi:10.1098/rsta.2015.0202, @https://lvdmaaten.github.io/publications/papers/JMLR_2008.pdf; @https://arxiv.org/abs/1802.03426] can help ‘compress’ information from a large number of features into a smaller number of features. 
 These techniques can be applied to characterize imaging data [@doi:10.1016/j.media.2020.101660], mass cytometry data [@doi:10.1038/ncomms14825], ‘omics data, and others. 
-These methods not only help in reducing the number of features, but can also be used to visualize structure or artifacts in the data (e.g. [@doi:10.1038/s41467-019-13056-x]), define subgroups within data (e.g. [@doi:10.1038/s41467-020-15351-4], or for feature selection/extraction during application of specific machine learning models[@doi:10.1007/978-3-030-03243-2_299-1].
+These methods not only help in reducing the number of features, but can also be used to visualize structure or artifacts in the data (e.g. [@doi:10.1038/s41467-019-13056-x]), define subgroups within data (e.g. [@doi:10.1038/s41467-020-15351-4], or for feature selection/extraction during application of specific machine learning models.[@doi:10.1007/978-3-030-03243-2_299-1]
```

Comment:
Should the period be before or after the citation? We're inconsistent it looks like.

Reply:
For superscript citations like Nature citation style I prefer to put the citation outside the punctuation (at least with a period...less sure about comma). This guide suggests always to put outside period or comma, though: https://libguides.usc.edu/c.php?g=293795&p=1956824

Nature Methods doesn't appear to have any specific guidance about this, however. 

What do you think?